lib/src/clojure_lsp/kondo.clj

```diff
@@ -123,17 +124,11 @@
 
     :java-class-definitions
     [(-> element
-         (dissoc :uri)
```

Comment:
I can't recall why I decided to dissoc uri, not sure if was only related to not store unused info on the db 🤔

Reply:
I think I remember seeing nil `:uri` being returned by clj-kondo? Maybe. I can't remember. Anyway, saving space or avoiding weird data could both be reasons we were dissocing uri. But now we add uri before calling this code, so we definitely don't want to dissoc it.